food-for-thought/components/Header.tsx

```diff
@@ -40,9 +40,13 @@ const { width } = Dimensions.get('window');
 
 const styles = StyleSheet.create({
     container: {
-        flex: 1,
-        flexDirection: 'row',
+        position: 'absolute',
+        top: 0,
```

Comment:
To help with pushing it down, I think `top: 20` will make sure its below the notch for iphone 14 users.

Reply:
Thank you so much, actioned.